src/core/unit.c

```diff
@@ -2463,12 +2461,6 @@ void unit_notify(Unit *u, UnitActiveState os, UnitActiveState ns, bool reload_su
         /* Some names are special */
```

Comment:
This comment can be removed.  The `if/else` block doesn't have any special names in it anymore.

Reply:
→ #8199.